### PR TITLE
Tests/csl811 header

### PR DESCRIPTION
**Pull request by rockbmb**
@gromakovsky Can you take a short look? It should be simple.

### CODE REVIEW

File: src/Pos/Block/Arbitrary.hs [hunk 1]
```diff
@@ -26,6 +28,7 @@ import qualified Pos.Types            as T
 import           Pos.Update.Arbitrary ()
 import           Pos.Util.Arbitrary   (makeSmall)
 import qualified Prelude
+import           System.Random        (mkStdGen, randomR)
```
Comment (gromakovsky) [inline src/Pos/Block/Arbitrary.hs]: Imports should be grouped according to style guide.

File: src/Pos/Block/Arbitrary.hs [hunk 2]
```diff
@@ -318,3 +321,68 @@ instance (Arbitrary (SscPayload ssc), SscHelpersClass ssc) =>
...
+                                then betweenZeroAndN (epochSlots - 1)
+                                else betweenXAndY s (epochSlots - 1)
+                            rndSlot = T.SlotId rndEpoch rndSlotIdx
+                        in Just rndSlot
+            params = T.VerifyHeaderParams
+                { T.vhpVerifyConsensus = consensus
+                , T.vhpPrevHeader = prev
+                , T.vhpNextHeader = next
+                , T.vhpCurrentSlot = randomSlotBeforeThisHeader
+                , T.vhpLeaders = Just $ fromList $ map T.addressHash thisHeadersEpoch
```
Comment (gromakovsky) [inline src/Pos/Block/Arbitrary.hs]: Why not use `nonEmpty` instead of `Just . fromList`? It avoids partial function.

**Comment by gromakovsky** [comment]
@rockbmb I left minor comments, you can merge it (don't forget to check that tests actually pass after merge), but you shouldn't work on `BlockSpec` now, because I expect it to be heavily refactored soon.

File: src/Pos/Block/Arbitrary.hs (hunk 2 above)
Comment (rockbmb) [inline src/Pos/Block/Arbitrary.hs]: I did not know about that function. Thanks for the heads-up.

**Pull request merged**